sqldelight-studio-plugin/src/main/kotlin/com/squareup/sqldelight/lang/SqlDelightFileViewProviderFactory.kt

```diff
@@ -43,6 +52,8 @@ internal class SqlDelightFileViewProvider(virtualFile: VirtualFile, language: La
     SingleRootFileViewProvider(psiManager, virtualFile, eventSystemEnabled, language) {
 
   val documentManager = PsiDocumentManager.getInstance(psiManager.project)
+  val file: SqliteFile
+    get() = getPsiInner(SqliteLanguage.INSTANCE) as SqliteFile
```

Comment:
Actually I guess the semantics are different. This calls each time it's accessed?